cellarium/ml/models/__init__.py

```diff
@@ -1,22 +1,26 @@
 # Copyright Contributors to the Cellarium project.
 # SPDX-License-Identifier: BSD-3-Clause
 
+from cellarium.ml.models.contrastive_mlp import ContrastiveMLP
 from cellarium.ml.models.geneformer import Geneformer
 from cellarium.ml.models.incremental_pca import IncrementalPCA
 from cellarium.ml.models.logistic_regression import LogisticRegression
 from cellarium.ml.models.model import CellariumModel, PredictMixin, ValidateMixin
 from cellarium.ml.models.mu_linear import MuLinear, abcdParameter
+from cellarium.ml.models.nt_xent import NT_Xent
```

Comment:
Refactor: I suggest having a `cellarium.ml.losses` folder and add our losses there.